packages/third-party/tailwind-preset/tsconfig.build.json

```diff
@@ -8,7 +8,14 @@
     "rootDir": "./src",
     "baseUrl": "./",
     "resolveJsonModule": true,
-    "isolatedDeclarations": false,
+    "declaration": true,
+    "emitDeclarationOnly": false,
+    "isolatedDeclarations": true,
+    "paths": {
+      "tailwindcss/types/config": ["node_modules/tailwindcss/types/config.d.ts"],
```

Comment:
I don't think you need this once change `tailwindcss/types/config` to `tailwindcss/types/config.js`.
```suggestion
```